css/styles.css

```diff
@@ -3,12 +3,166 @@
 }
 
 :root{
-
+    --primary-0:#FFFFFF;
```

Comment:
I think we should use SCSS instead of plain CSS.

Something like:
https://blog.devgenius.io/use-sass-on-your-plain-html-css-js-projects-4f09de547f8a

You can read more about the benefits of SCSS. You can read some quick walkthrough to SCSS that explains some advantages over CSS. Then you can keep learning side by side. It's a superset of CSS.